src/main/java/it/cnr/isti/hpc/wikipedia/parser/ArticleParser.java

```diff
@@ -231,6 +231,8 @@ private void setRedirect(Article article, String mediawiki) {
 	private void setTables(Article article, ParsedPage page) {
 		List<Table> tables = new ArrayList<Table>();
 
+		List<Link> links = new ArrayList<Link>();
```

Comment:
If you set a javac >=1.7 it should like that. I started to use it, and it is happy :)